CHANGELOG.md

```diff
@@ -8,30 +8,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-- Show "You are here" marker in webui.
+- Implement stateful failover mode.
```

Comment:
I propose to add a link to documentation where user could obtain necessary details about it.